extensions/wikia/RWEPageHeader/scripts/rwe-page-header.js

```diff
@@ -19,5 +19,44 @@ require(['wikia.window', 'jquery', 'wikia.tracker'], function (window, $, tracke
 		$('.rwe-page-header-nav__element-dropdown > .rwe-page-header-nav__link').on('click', function (e) {
 			e.preventDefault();
 		});
+
+
+
+		var $wikiaTopAds = $('.WikiaTopAds');
+		var $globalNav = $('.wds-global-navigation-wrapper');
+
+
+		window.BannerNotification.prototype.onShow = window.BannerNotification.prototype.show;
+		window.BannerNotification.prototype.onHide = window.BannerNotification.prototype.hide;
+
+		function setNotificationPosition() {
+			var $globalNav = $('.wds-global-navigation-wrapper');
+			var $notificationsWrapper = $('.banner-notifications-wrapper');
+
+			$notificationsWrapper.css('top', ($globalNav.position().top + 55) + 'px');
+
+			if ($globalNav.position().top < 10) {
+				var topMargin = $notificationsWrapper.height();
+				$wikiaTopAds.css('marginTop', topMargin + 'px');
+			}
+		}
+
+		window.BannerNotification.prototype.show = function() {
+			this.onShow();
+			var $notificationsWrapper = $('.banner-notifications-wrapper');
+			$notificationsWrapper.detach();
+
+			$notificationsWrapper.insertAfter($globalNav);
+			// setNotificationPosition();
+
+			return this;
+		};
+
+		window.BannerNotification.prototype.hide = function() {
+			this.onHide();
+			// setNotificationPosition();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)